ZenPacks/zenoss/ZenPackLib/lib/params/EventClassSpecParams.py

```diff
@@ -9,6 +9,7 @@
 
 from .OrganizerSpecParams import OrganizerSpecParams
 from .EventClassMappingSpecParams import EventClassMappingSpecParams
+from .SpecParams import SpecParams
```

Comment:
I don’t see why `SpecParams.__init__(self)` is called in those cases instead of `OrganizerSpecParams.__init__(self)`. Going to the grandparent instead of parent seems questionable.